docs/advanced.rst

```diff
@@ -134,7 +134,7 @@ Transport Layer
 ---------------
```

Comment:
Remove this section about transports. This information is already in advance configuration of pubsub.rst.